### PR TITLE
Fix error in "compute vocabulary" for protos, add test of proto vocabulary computation

### PR DESCRIPTION
**Pull request by timburks**
This will probably hurt our code coverage temporarily because it brings "compute vocabulary" into scope and only tests the proto vocabulary computation. (I will follow up with more test PRs)

### CODE REVIEW

File: cmd/registry/cmd/compute/vocabulary/vocabulary.go [hunk 1]
```diff
@@ -100,6 +100,10 @@ func Command() *cobra.Command {
 			}
 		},
 	}
+	cmd.PersistentFlags().String("filter", "", "Filter selected resources")
+	cmd.PersistentFlags().Bool("dry-run", false, "if set, computation results will only be printed and will not stored in the registry")
+	cmd.PersistentFlags().Int("jobs", 10, "Number of actions to perform concurrently")
```
Comment (timburks) [inline cmd/registry/cmd/compute/vocabulary/vocabulary.go:105]: If these aren't here, the flags are unavailable when we instantiate the test command.

**Comment by theganyo** [comment]
There is a lot here and I'm unsure what the original error was. Maybe we should walk through this.

File: cmd/registry/cmd/compute/vocabulary/vocabulary-proto.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func NewVocabularyFromZippedProtos(b []byte) (*metrics.Vocabulary, error) {
 	}
 
 	for _, f := range r.File {
-		if strings.HasSuffix(f.Name, ".proto") {
+		if !strings.HasSuffix(f.Name, ".proto") {
```
Comment (timburks) [inline cmd/registry/cmd/compute/vocabulary/vocabulary-proto.go:44]: @theganyo This is the error -- introduced in a big refactor a while ago, it was preventing the proto vocabulary computation from finding anything.

**Review by theganyo — APPROVED** [review]
I see, nearly all of the files are `testdata`.

**Pull request merged**